src/Umbraco.Web.BackOffice/Filters/MemberSaveModelValidator.cs

```diff
@@ -179,6 +184,11 @@ internal bool ValidateUniqueLogin(MemberSave model)
 
     internal bool ValidateUniqueEmail(MemberSave model)
     {
+        if(_securitySettings.RequireUniqueEmailForMembers == false)
+        {
+            return true;
+        }
```

Comment:
This check should be moved to the `ValidatePropertiesData()` method, so this `ValidateUniqueEmail()` keeps doing what you expect it to do (based on the method name) and is only called when you require unique emails.

Reply:
I see your point. I have moved the conditional check to the `ValidatePropertiesData()` method. I am not in loving with my change. I did it as a separate conditional statement, but the number of conditional statements in this method is become a lot but the trade off is that the current `validEmail` variable check would become more complex.  i.e. drop my conditional on `_securitySettings.MemberRequireUniqueEmail` and change `if(validEmail == false) { ...` to `if (_securitySettings.MemberRequireUniqueEmail && ValidateUniqueEmail(model) == false) { ... `. Thoughts? 

New to committing to Umbraco so not sure if there is a preference for verbose easier to read code or more concise code.

Reply:
Well, I changes my code to the more concise version. Without major class refactoring I am not sure how to make CodeScene happy.

Reply:
Need help to know if there are actions I need to take? Builds local, but fails some check on the GitHub pipeline that I need help understanding. Also would like to know if this will be considered for inclusion. Thank you.